static/js/components/EmailCompositionDialog.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+import R from 'ramda';
+import type { EMAIL_EDIT_STATE } from '../reducers/email';
+
+const createDialogActions = (close, send) => ([
+  <Button
+    type="button"
+    className="dialog-button cancel-button"
+    key="first"
+    onClick={close}
+  >
+    Cancel
+  </Button>,
+  <Button
+    type="button"
+    className="dialog-button save-button"
+    key="second"
+    onClick={send}
+  >
+    Send
+  </Button>
+]);
+
+const hitsCount = sk => R.isNil(sk) ? 0 : sk.getHitsCount();
```

Comment:
1. nitpick: can we just change `sk` to `searchkit` or `searchkit_prop`?
1. general question: i'm seeing a lot of `R` use lately. it seems like theres a good amount of overlap between R and lodash, this `isNil` method being one of them (https://lodash.com/docs#isNil). does R have any specific advantages? i ask b/c i think it's safe to say that lodash/underscore are more widely used/understood than R. there's a bit more cognitive load when an uninitiated dev (let's call him 'davin') sees a method being invoked on R